frontend/device/kobo/device.lua

```diff
@@ -803,6 +853,14 @@ function Kobo:toggleChargingLED(toggle)
     if toggle == true then
         -- NOTE: Technically, Nickel forces a toggle off before that, too.
         --       But since we do that on startup, it shouldn't be necessary here...
+        if self:isSunxi() then
+            f:write("ch 3")
+            f:flush()
+            f:write("cur 1")
+            f:flush()
+            f:write("dc 63")
```

Comment:
Some comments about what each of these commands do (if you could guess) ?

Reply:
Done ;).